packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -109,8 +110,8 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     return SCHEMA_CRUD_PLUGIN_NAME;
   }
 
-  protected buildSchemaForModels(schemaComposer: SchemaComposer<any>, models: ModelDefinition[]) {
-    this.createStandardTypes(schemaComposer)
+  protected buildSchemaForModels(schemaComposer: SchemaComposer<any>, models: ModelDefinition[], schema: GraphQLSchema) {
+    this.createStandardInputTypes(schemaComposer, schema);
```

Comment:
This method can probably be renamed, since we are also generating dynamic input types based on unknown scalars.

Reply:
Good idea. Do you have any naming suggestion for inspiration ? ;-)

Reply:
Hmm. `createSchemaCRUDTypes`? Since it creates the various types for CRUD like pagination, input types, enums and so on.